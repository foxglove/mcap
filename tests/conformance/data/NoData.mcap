version https://git-lfs.github.com/spec/v1
oid sha256:555cd5557a34db8f780028386ea5fd5abb5f3e90896b6f261475907dc0da7462
size 62
