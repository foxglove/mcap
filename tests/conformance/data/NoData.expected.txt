Header profile="" library=""
Footer summaryStart=25 summaryOffsetStart=0
